docs/reference/content/getting-started/quick-tour.md

```diff
@@ -0,0 +1,206 @@
+`Employee` instances will be saved in to the `employees` collection instead.  There is a little more to this annotation but the 
+[annotations guide]({{< relref "guides/annotations.md#entity" >}}) covers those details.
+
+The `@Indexes` annotation lists which annotations Morphia should create.  In this instance, we're defining an index named `salary` on the
+ field salary with the default ordering of ascending.  More information on indexing can found
+  [here]({{< relref "guides/annotations.md#indexes" >}}).
+ 
+We've marked the `id` field to be used as our primary key (the `_id` field in the document).  In this instance we're using the Java driver 
+type of `ObjectId` as the ID type.  The ID can be any type you'd like but is generally something like `ObjectId` or `Long`.  There are 
+two other annotations to cover but it should be pointed out now that other than transient and static fields, Morphia will attempt copy 
+every field's value in to a document bound for the database.
+
+The simplest of the two remaining annotations is `@Property`.  This annotation is entirely optional.  If you leave this annotation off, 
+Morphia will use the Java field name as the document field name.  Often times this is fine.  However, some times you'll want to change 
+the document field name for any number of reasons.  In those cases, you can use `@Property` and pass it the name to be used when this 
+class is serialized out to a document to be handed off to mongodb.  
+
+This just leaves `@Reference`.  This annotation is telling Morphia that this field refers to other Morphia mapped entities.  In this case 
+Morphia will store what mongodb calls a `DBRef` which is just a collection name and key value.  These referenced entities must already be
+ saved or at least have an ID assigned or Morphia will throw an exception.
+ 
+## Saving Data
+
+For the most part, you treat your Java objects just like you normally would.  When you're ready to write an object to the database, it's 
+as simple as this:
+
+```java
+final Employee elmer = new Employee("Elmer Fudd", 50000.0);
+datastore.save(elmer);
+```
+
+Taking it one step further, lets define some relationships and save those, too.
+
+```java
+final Employee daffy = new Employee("Daffy Duck", 40000.0);
+datastore.save(daffy);
+
+final Employee pepe = new Employee("Pepé Lepew", 25000.0);
+datastore.save(pepe);
+
+elmer.getDirectReports().add(daffy);
+elmer.getDirectReports().add(pepe);
+
+datastore.save(elmer);
+```
+
+As you can see, we just need to create and save the other Employees then we can add them to the direct reports list and 
+save.  Morphia takes care of saving the keys in Elmer's document that refer to Daffy and Pepé.  Updating data in mongodb is as simple as 
+updating your Java objects and then calling `datastore.save()` with them again.  For bulk updates (everyone gets a raise!) this is not 
```

Comment:
It's not just the "everyone gets a raise" scenario.  It's also regular updates, like pushing a value onto an array, that can be done atomically without a read-modify-write sequence.